### PR TITLE
feat: adding GA to init script

### PR DESCRIPTION
**Pull request by kobenguyent**
## Motivation/Description of the PR
Screen Shot 2022-09-28 at 11 20 48 AM

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

**Review by DavertMik — CHANGES_REQUESTED** [review]
1. this is possible only for Playwright / Puppeteer and maybe TestCafe as webdriver will require more sophisticated setup  
2. if a Playwright option is selected a command to install Playwright should be added  

```
    - name: Install deps
      run: npx playwright install-deps
```

if a user selects WebDriver or Appium - do not propose to create this file. As it won't be working

File: lib/command/init.js [hunk 1]
```diff
@@ -245,6 +254,33 @@ module.exports = function (initPath) {
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npx playwright install-deps
+      - run: npm i
+      - run: npm run test
```
Comment (DavertMik) [inline lib/command/init.js]: I would prefer to use `npx codeceptjs run` here to ensure that CodeceptJS is executed

File: lib/command/init.js [hunk 2]
```diff
@@ -245,6 +254,33 @@ module.exports = function (initPath) {
         }
       }
 
+      if (installGitHubActions === true) {
+        const ghaYml = `name: Run Tests
```
Comment (DavertMik) [inline lib/command/init.js]: can we move this to the header of the file?

File: lib/command/init.js [hunk 3]
```diff
@@ -107,6 +108,13 @@ module.exports = function (initPath) {
       choices: helpers,
       message: 'What helpers do you want to use?',
     },
+    {
+      type: 'confirm',
+      name: 'installGitHubActions',
+      message: 'Add a GitHub Actions workflow?',
+      default: false,
+      when: (answers) => ['Playwright', 'Puppeteer'].includes(answers.helper) === true,
```
Comment (DavertMik) [inline lib/command/init.js:117]: I don't think this will work for Puppeteer

**Review by DavertMik — CHANGES_REQUESTED** [review]

File: lib/command/init.js (hunk 3 above)
Comment (kobenguyent) [reply lib/command/init.js:117]: https://github.com/codeceptjs/CodeceptJS/blob/3.x/.github/workflows/puppeteer.yml

I guess this shall work cause checking this file not any additional libs should be installed for puppeteer.

File: lib/command/init.js (hunk 2 above)
Comment (kobenguyent) [reply lib/command/init.js]: are you mentioning that moving this step?

File: lib/command/init.js (hunk 3 above)
Comment (DavertMik) [reply lib/command/init.js:117]: But we have 

```
npx playwright install-deps
```

**Comment by DavertMik** [comment]
Updating documentation on initialization got me feeling that init script is already overfilled with lots of questions and it would be hard to answer all of them from start

Maybe we could move CI setup into other script? For instance in `generate.js`
So please @PeterNgTr if you want to keep it, move it to this file

Actually, you don't need CI setup until you have a set of tests written
So this probably should not be part of initial setup

**Pull request closed without merging**